src/python/pants/rules/core/test.py

```diff
@@ -66,7 +72,12 @@ def coordinator_of_tests(target):
   #if isinstance(target.adaptor, PythonTestsAdaptor):
   # See https://github.com/pantsbuild/pants/issues/4535
   if target.adaptor.type_alias == 'python_tests':
+    logger.info("Starting test for {}...".format(target.address.reference()))
     result = yield Get(PyTestResult, HydratedTarget, target)
+    logger.info("Result of test for {}: {}".format(
```

Comment:
Would prefix the result value before the target name so that it renders at a consistent location.